Kconfig.nrf

```diff
@@ -70,6 +70,13 @@ config NORDIC_QSPI_NOR
 
 DT_ZEPHYR_FLASH := zephyr,flash
 DT_CHOSEN_ZEPHYR_FLASH := $(dt_chosen_path,$(DT_ZEPHYR_FLASH))
+
+config FLASH_LAYOUT_PAGE_SIZE
```

Comment:
right below we have `NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE` which has the exact same default value https://github.com/nrfconnect/sdk-nrf/blob/dd6e053db71febe1fa8aefd924b0ce7534886f6a/Kconfig.nrf#L80-L81

Do we really need both settings, and why ?